api/pkg/apis/v1alpha1/providers/target/k8s/k8s.go

```diff
@@ -91,6 +97,23 @@ func K8sTargetProviderConfigFromMap(properties map[string]string) (K8sTargetProv
 	if v, ok := properties["context"]; ok {
 		ret.Context = v
 	}
+	if v, ok := properties["timeout"]; ok {
+		ret.Timeout = v
+	} else {
+		ret.Timeout = "5m" // default 5 mins
+	}
+	if v, ok := properties["noWait"]; ok {
+		val := v
+		if val != "" {
+			bVal, err := strconv.ParseBool(val)
+			if err != nil {
+				return ret, v1alpha2.NewCOAError(err, "invalid bool value in the 'wait' setting of K8s reference provider", v1alpha2.BadConfig)
```

Comment:
invalid bool value in the 'noWait' setting of K8s target provider

Reply:
Same as line 122, we have a typo: it is K8s target provider. Please modify as well.

Reply:
fixed